tests/test_exec_nodes.py

```diff
@@ -157,9 +157,5 @@ def test_call_directly_with_ignore() -> None:
 
     cfg.TAWAZI_EXECNODE_OUTSIDE_DAG_BEHAVIOR = XNOutsideDAGCall.ignore
 
-    with pytest.warns(None) as record:  # type: ignore[call-overload]
```

Comment:
doesn't this expect a warning ?  why remove it!?

Reply:
because it doesn't raise one hhh

Reply:
and also pytest.warns(None) is deprecated and raises an error in the CI